acceptance/auth/basic/out.requests.txt

```diff
@@ -0,0 +1 @@
+{"headers":{"Authorization":["Basic dXNlcm5hbWU6cGFzc3dvcmQ="],"User-Agent":["cli/[DEV_VERSION] databricks-sdk-go/[SDK_VERSION] go/[GO_VERSION] os/[OS] cmd/current-user_me cmd-exec-id/[UUID] auth/basic"]},"method":"GET","path":"/api/2.0/preview/scim/v2/Me","body":""}
```

Comment:
Could send a separate PR with this output pretty-printed? I believe nothing would break, jq can still process is as a stream.

Reply:
done in this PR.